nodejs/week1/homework/src/backend/app.js

```diff
@@ -0,0 +1,70 @@
+const express = require("express");
+const app = express();
+module.exports = app;
+// import data here
+const meals = require("./data/meals");
+const reservations = require("./data/reservations");
+const reviews = require("./data/reviews");
+const mealsNum = meals.map((meal) => meal.id);
+const reviewMealId = reviews.map((review) => review.mealId);
+// this is where you will be adding your routes
+app.get("/", async (request, response) => {
+  response.send("asd");
+});
+// meals with review
```

Comment:
Nitpick(comments):
Most of these comments just repeat the route name, that is usually enough information.
If you also extract the route handler and give that function a name you can put more context into that name:
`app.get('/meals', listMealsWithReviews)`